interface/portraits/portraits_highelffel_sprites.gfx

```diff
@@ -0,0 +1,242 @@
+# hair, mouth, chin, nose, eyes, cheek, ear, neck
+spriteTypes = {
+
+############## corrupted highelf ##########################
+
+	spriteType = {
+		name = "PORTRAIT_highelffelfgfx_child_male"
+		texturefile = "gfx\\characters\\highelf_child\\bloodelf_child_male.dds"
+		noOfFrames = 1
+		norefcount = yes
+	}
+
+	spriteType = {
+		name = "PORTRAIT_highelffelgfx_child_female"
+		texturefile = "gfx\\characters\\highelf_child\\bloodelf_child_female.dds"
+		noOfFrames = 1
+		norefcount = yes
+	}
```

Comment:
Better put child sprite declarations in `portraits_highelf_sprites` (at the top right after the high elven children).